ports/rp2/main.c

```diff
@@ -59,6 +59,23 @@
 #include "lib/cyw43-driver/src/cyw43.h"
 #endif
 
+#if MICROPY_HW_USB_VENDOR
+#include "bsp/board.h"
+#include "tusb_port.h"
+
+#include "hardware/watchdog.h"
+
+#include "main.h"
+#endif
+
+#ifndef MICROPY_GC_HEAP_SIZE
```

Comment:
This is inadvertently reverting https://github.com/micropython/micropython/commit/c80e7c14e6305e50e3b39f97172d4d8fe1214d3b

Reply:
This was leftover from a previous approach I was trying. It has been removed now.